api/tests/services/test_auth_service.py

```diff
@@ -1,19 +1,60 @@
 from app.models.user import User
 from app import db
 
-def test_helloworld():
-    assert True == True
-
-def test_create_user(client):
-    with client.application.app_context():
-        # Create a new user
-        new_user = User(username='test_user', password='bad_password')
-        db.session.add(new_user)
-        db.session.commit()
-
-        # Query the user from the database
-        user_from_db = User.query.filter_by(username='test_user').first()
-
-        # Check that the added properties match
-        assert user_from_db.username == new_user.username
-        assert user_from_db.password == new_user.password
+# SAMPLE CODE FOR INTERACTING WITH DATABASE
+# def test_create_user(client):
+#     with client.application.app_context():
+#         # Create a new user
+#         new_user = User(username='test_user', password='bad_password')
+#         db.session.add(new_user)
+#         db.session.commit()
+
+#         # Query the user from the database
+#         user_from_db = User.query.filter_by(username='test_user').first()
+
+#         # Check that the added properties match
+#         assert user_from_db.username == new_user.username
+#         assert user_from_db.password == new_user.password
+
+class TestLogin:
```

Comment:
Looks good to me but I think the code needs to be completed with actual implementation of the login and registration functionality and the test methods that assert specific conditions to ensure that the functionality works as expected.

Reply:
For sure! The login/registration code for the api is in `api/routes/auth.py` and `api/services/auth_service.py`
The code for the frontend is in `src/views/Login.Vue` and `src/views/Register.vue`

Both are completed and should have changes showing in this PR